lib/begin.go

```diff
@@ -188,6 +189,22 @@ func (a *ACBuild) writeEmptyManifest() error {
 		return err
 	}
 
+	archvalue := runtime.GOARCH
+	if runtime.GOOS == "linux" && archvalue == "arm" {
+		archvalue = "armv7l"
+		walker := func(path string, info os.FileInfo, err error) error {
```

Comment:
What's this doing? Looking at the acbuild binary that's in memory for some specific value?